chain/chain/src/chain.rs

```diff
@@ -357,7 +361,7 @@ impl Chain {
         self.doomslug_threshold_mode = DoomslugThresholdMode::NoApprovals
     }
 
-    pub fn compute_bp_hash_inner(bps: Vec<ValidatorStake>) -> Result<CryptoHash, Error> {
+    pub fn compute_bp_hash_inner<T: BorshSerialize>(bps: Vec<T>) -> Result<CryptoHash, Error> {
```

Comment:
If you make this change then the function name doesn't make sense any more. We should either just get rid of this function or rename it and move to utils

Reply:
Done in https://github.com/near/nearcore/pull/4115/commits/e62cbf26d10b9d19bcc30e07dab5c0fe4ee76b20

Though I did not move the function. Since the error type is specific to `Chain`, I thought it made sense to leave it where it was.